src/common/constants.ts

```diff
@@ -37,6 +37,7 @@ const getDomainAppID = () => {
 }
 
 export const eu_domains = ['eu', 'staging-eu']
+export const eu_urls = ['eu.deriv.com', 'staging-eu.deriv.com']
```

Comment:
const deriv_com_url = 'deriv.com'
const deriv_me_url = 'deriv.me'
const deriv_be_url = 'deriv.be'

Reply:
Can help me on this @habib-deriv ? 
This line is the same as master branch. do I need to update it in this branch? 
thanks.

Reply:
Okay, let's keep it as it is for now